aten/src/ATen/native/native_functions.yaml

```diff
@@ -5523,13 +5523,13 @@
 
 - func: pow_.Scalar(Tensor(a!) self, Scalar exponent) -> Tensor(a!)
   use_c10_dispatcher: full
-  variants: method
+  variants: function, method
```

Comment:
Instead of changing `pow_` to also be a function you can use `base.pow_` in Pow.cpp. For example:

```
return at::float_power_(base, c10::scalar_to_tensor(exp, base.device()));
```

can become

```
return base.pow_(c10::scalar_to_tensor(exp, base.device()));
```

Reply:
Wow, that's really convenient, thanks for reminding me!